CommunityGuidelines.md

```diff
@@ -15,7 +15,9 @@ The Mozilla community is for people who are interested in the Mozilla project an
 
 - **Be nice**: we are all part of the same community, so be friendly, be welcoming, and generally be a nice person. Be someone that other people want to be around.
 
-- **Be respectful and constructive**: We're a diverse group (from all over Tunisia!), so we're not going to agree all the time. Remember to be respectful and constructive with your communication to your fellow Mozillians. Don't get into fights, don't make personal attacks, don't vent or rant unconstructively. __Everyone__ should take responsibility for the community and take the initiative to diffuse tension and stop a potential flamewar, fight or negative thread _as early as possible_.
+- **Be respectful**: We're a diverse group (from all over Tunisia!), so we're not going to agree all the time. Remember to be respectful and constructive with your communication to your fellow Mozillians. Don't get into fights, don't make personal attacks, don't vent or rant unconstructively.
+
+- **Be constructive**: __Everyone__ should take responsibility for the community. Take the initiative to diffuse tension and stop a potential flamewar, fight or negative thread _as early as possible_. No one has the right to share our internal misunderstanding in public.
```

Comment:
`Take the initiative to diffuse tension and stop a potential flamewar, fight or negative thread _as early as possible_.` doesn't look like it falls in the "constructive" basket but I think we can let it slide.

`No one has the right to share our internal misunderstanding in public.`: probably you'd want to lower the tone a little? Something like `If there is a misunderstanding or an issue going on, we invite you to avoid communicating it publicly or on social networks, and if you feel like the situation is reaching a deadend and no one in the community is showing good intentions, have a staff member that you feel comfortable talking to about the matter give you an advice on how to handle it.`